app/controllers/comments_controller.rb

```diff
@@ -206,19 +206,12 @@ def load_commentable
   end
 
   def index
-    if !@commentable.nil?
-      @comments = @commentable.comments.reviewed.page(params[:page])
-      if @commentable.class == Comment
-        # we link to the parent object at the top
-        @commentable = @commentable.ultimate_parent
-      end
-    else
-      if logged_in_as_admin?
-        @comments = Comment.top_level.not_deleted.limit(ArchiveConfig.ITEMS_PER_PAGE).ordered_by_date.include_pseud.select { |c| c.ultimate_parent.respond_to?(:visible?) && c.ultimate_parent.visible?(current_user) }
-      else
-        redirect_back_or_default(root_path)
-        flash[:error] = ts("Sorry, you don't have permission to access that page.")
-      end
+    return redirect_to "/404" if @commentable.blank?
```

Comment:
To give the 404, let's call this method:

https://github.com/otwcode/otwarchive/blob/c981777f3983b5d114f12dc6757542cbe4535cb3/app/controllers/application_controller.rb#L25-L27

It's just a little neater, and if we ever want to change the URL for some reason, we only need to update it in one place.

Reply:
done! thanks :D